model/build.gradle

```diff
@@ -4,8 +4,11 @@ apply plugin: 'com.google.protobuf'
 protobuf {
   protoc {
     // To build protoc in M1 mac. For context, see: #3912.
-    if (project.rootProject.hasProperty('protobuf_platform')) {
-      artifact = "com.google.protobuf:protoc:3.8.0:${project.rootProject.property("protobuf_platform")}"
+    if (osdetector.os == "osx") {
+      Properties properties = new Properties()
+      properties.load(project.rootProject.file("local.properties").newDataInputStream())
```

Comment:
Does the stream also need to be closed to avoid potential resource leaks?

Reply:
Yes, that's a good point. Is there need to enclose this in a try-catch block? I'm not so certain whether we want those in a gradle script file.

Reply:
Yes, that's a good point. Is there need to enclose this in a try-catch block? I'm not so certain whether we want those in a gradle script file.